sugoi-api-ldap-store-provider/src/main/java/fr/insee/sugoi/store/ldap/LdapStoreBeans.java

```diff
@@ -45,6 +46,12 @@
   @Value("${fr.insee.sugoi.ldap.default.group_source_pattern:}")
   private String defaultGroupSourcePattern;
 
+  @Value("${fr.insee.sugoi.ldap.default.app_managed_attribute_key:}")
+  private String defaultAppManagedAttributeKey;
+
+  @Value("${fr.insee.sugoi.ldap.default.app_managed_attribute_pattern:}")
```

Comment:
it will throw a new AppCannotManagedAttributeException( "Cannot add attribute to user: app cannot managed " + attributeKey); which lead to a 401 request i believe

Reply:
for the moment it throws a null pointer exception

Reply:
resolved throw 403 forbidden if not set